src/styles/main.scss

```diff
@@ -1 +1,65 @@
-// put code here
+body {
+  margin: 0;
+}
+
+.watch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 3px dotted black;
+  border-radius: 50%;
+  position: relative;
+  margin: 50px;
+
+  &__minute {
+    width: 15px;
+    height: 100px;
+    background-color: blue;
+    position: absolute;
+    top: 24.4vmin;
```

Comment:
![image](https://github.com/mate-academy/layout_stop-watch/assets/86923089/afbca400-4ce5-494e-a3ef-3a4fd49c9ead)

use relative values instead

fix arrows and dot position